src/jspm.conf.js

```diff
@@ -205,9 +205,21 @@ System.config({
       "crypto": "github:jspm/nodelibs-crypto@0.1.0",
       "url": "github:jspm/nodelibs-url@0.1.0"
     },
+    "npm:babel-polyfill@6.16.0": {
```

Comment:
@vblagoje I think we need all of them as the changes further down are defining what `babel-polyfill` depends on.